R/remote_read.R

```diff
@@ -0,0 +1,29 @@
+#` `remote_read.R`
+#` @description Utilities that read in remote sources of data directly into the R environment.
+
+#' Read in a file in table formate from a remote source.
+#'
+#' Currently this function only works with .csv and .txt files.
+#'
+#' @param url character.  URL to the file to import into R, it must be write-enabled for the user.
+#' @param ... additional arguments that may be passed  \code{read.table}
+#' @author Kalyn R. Dorheim (kalyn.dorheim@pnnl.gov)
+#' @export
+remote_read <- function(url, ...){
+
+  # Check the url input
+  assertthat::assert_that(grepl('.txt|.csv', url), msg = 'this function only supports data table import for files saved as a .txt or .csv')
+  assertthat::assert_that(grepl('download=1', url), msg = 'url incorrectly formatted, it must end with "download=1"')
```

Comment:
I would remove this assertion because the `download=1` label is likely unique to Zenodo.  If we are downloading from another service, this will likely fail.

Reply:
Ah! good call didn't realize that was a zenodo specific addressed with commit f5664d1